.gitignore

```diff
@@ -0,0 +1,6 @@
+.Rproj.user
+.Rhistory
+.RData
+.Ruserdata
+*.Rproj
```

Comment:
We've tended to try to not put anything extraneous on github as much as possible is probably the original reason. Also, people tend to name their projects whatever they want, so this would start to force people to name them all the same name.